src/GeneralStateTestsFiller/stSystemOperationsTest/doubleSelfdestructTest2Filler.yml

```diff
@@ -0,0 +1,67 @@
+doubleSelfdestructTest2:
+  info:
```

Comment:
_info

Reply:
I'll fix.

Reply:
Suddenly I realize that the schema checking doesn't work on YAML.  I created an issue https://github.com/ethereum/tests/issues/456